test/controllers/api/v2/provisioning_templates_controller_test.rb

```diff
@@ -126,7 +126,11 @@ class Api::V2::ProvisioningTemplatesControllerTest < ActionController::TestCase
     assert_response :success
     assert_equal 'text/plain', response.content_type
     assert_equal templates(:pxekickstart).to_erb, response.body
-    assert_equal 'attachment; filename="centos5_3_pxelinux.erb"', response.headers['Content-Disposition']
+    if Rails.version.start_with?('5.2')
+      assert_equal 'attachment; filename="centos5_3_pxelinux.erb"', response.headers['Content-Disposition']
+    else
+      assert_equal 'attachment; filename="centos5_3_pxelinux.erb"; filename*=UTF-8\'\'centos5_3_pxelinux.erb', response.headers['Content-Disposition']
```

Comment:
is there a point to assert exact equality? what if we 
```suggestion
      assert_match /attachment; filename="centos5_3_pxelinux.erb"/
```
we don't need the if then :thinking: